dbt/adapters/athena/connections.py

```diff
@@ -74,6 +75,10 @@ def type(self) -> str:
     def unique_field(self) -> str:
         return f"athena-{hashlib.md5(self.s3_staging_dir.encode()).hexdigest()}"
 
+    @property
+    def get_effective_num_retries(self) -> int:
+        return self.num_boto3_retries if self.num_boto3_retries is not None else self.num_retries
```

Comment:
```suggestion
        return self.num_boto3_retries or self.num_retries
```
we can simplify this a bit if you don't mind